src/main/java/com/albert/rpsgame/Move.java

```diff
@@ -0,0 +1,44 @@
+package com.albert.rpsgame;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public enum Move {
+    ROCK(1), PAPER(2), SCISSORS(3), LIZARD(4), SPOCK(5);
+
+    Move(int i) {
```

Comment:
Po co ten pusty konstruktor? :)

Reply:
Bez niego nie mogę do każdego ruchu przypisać indeksu w okrągłym nawiasie. Bez takiego konstruktora się nie da. :(

Reply:
Dobra, nie było tematu, właśnie wpadłem jak to zrobić inaczej :D